lib/Red/Driver/Pg.pm6

```diff
@@ -34,6 +34,22 @@ multi method translate(Red::AST::Select $_, $context?, :$gambi where !*.defined)
     self.Red::Driver::CommonSQL::translate($_, $context, :gambi);
 }
 
+multi method translate(Red::AST::In $_, $context?) {
```

Comment:
Isn't it common? Couldn't it be on CommonSQL?

Reply:
It's common, But It's strange that it works in SQLite.